### PR TITLE
fix image src, add release steps and more badges

### PR DESCRIPTION
**Pull request by circlecube**
## Proposed changes

This adds the steps we discussed today to a quick list on the main plugin readme file. Should be easy to follow them for releases in the future. I tried to add any relevant links to point future us in the right direction in case we run into any issues. Please read through them to make sure I didn't forget anything. Preview in branch: https://github.com/bluehost/bluehost-wordpress-plugin/tree/add/release-notes

I also updated the logo image since the old one was broken and added more badges so we can better see status of the latest build, lint and tests.

The only issue I see is that in an .md file like this, the checkboxes are not interactive, so not much point in having them here. It was a nice thought though.

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by aalani95 — APPROVED** [review]
Perfect. Approved!

**Review by aalani95 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
+  - [ ] The [plugin header version](bluehost-wordpress-plugin.php#L5).
+  - [ ] The [plugin constant version](bluehost-wordpress-plugin.php#L35).
+- [ ] Ensure `master` branch has passing cypress tests.
+- [ ] Ensure `master` branch has passing lint tests.
+- [ ] Ping team via chat and announce latest build available for testing.
+- [ ] Download latest build and install on a site for a manual test.
+- [ ] Confirm no issues are found in testing.
+- [ ] Create a [new release](https://github.com/bluehost/bluehost-wordpress-plugin/releases/new) tagged and named for the version.
+- [ ] Ensure the satis [build](https://bluehost.github.io/satis/#bluehost/bluehost-wordpress-plugin) is [triggered](https://github.com/bluehost/bluehost-wordpress-plugin/actions/workflows/satis-webhook.yml) and [completes](https://github.com/bluehost/satis/actions).
+- [ ] Ensure [cloudflare worker](https://wp-github-release-api.bluehost.workers.dev/plugins/bluehost/bluehost-wordpress-plugin) displays the release as latest/current version.
```
Comment (wpscholar) [inline README.md:32]:
```suggestion
- [ ] Ensure [cloudflare worker](https://hiive.cloud/workers/release-api/plugins/bluehost/bluehost-wordpress-plugin/) displays the release as latest/current version.
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+- [ ] Ping team via chat to announce end of release process.
+- [ ] Watch for plugin release rollout in hiive.
```
Comment (wpscholar) [inline README.md:34]:
```suggestion
- [ ] Watch for plugin release rollout in Hiive.
```

**Review by wpscholar — CHANGES_REQUESTED** [review]
We should update the URL reference since we'll be getting that updated shortly.

**Pull request merged**